src/iOS/Xamarin.iOS/Resources/licenses.md

```diff
@@ -0,0 +1,96 @@
+﻿# Licenses for 3rd-party software packages
+
+The ArcGIS Runtime SDK for .NET sample viewer (iOS version) was built using several free and open source software libraries and resources. 
+
+## MarkedNET
+
+This app uses MarkedNET under the MIT license. It is used to render the content on the sample description page.
+
+[Project on GitHub](https://github.com/T-Alex/MarkedNet) | [Project on NuGet](https://www.nuget.org/packages/MarkedNet/) | [License](https://github.com/T-Alex/MarkedNet/blob/master/LICENSE.md)
+
+MIT license reproduced in full:
+
+------
+
+The MIT License (MIT)
+
+Copyright (c) 2015 Alex Titarenko, Christopher Jeffrey
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+----
+
+## Highlight.JS
+
+This app uses highlight.js under the BSD license to provide syntax highlighting on the sample source code pages.
+
+[Project on GitHub](https://github.com/highlightjs/highlight.js) | [License](https://github.com/highlightjs/highlight.js/blob/master/LICENSE)
+
+BSD license reproduced in full:
+
+---
+
+Copyright (c) 2006, Ivan Sagalaev
+All rights reserved.
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright
+    notice, this list of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the above copyright
+    notice, this list of conditions and the following disclaimer in the
+    documentation and/or other materials provided with the distribution.
+* Neither the name of highlight.js nor the names of its contributors 
+    may be used to endorse or promote products derived from this software 
+    without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE REGENTS AND CONTRIBUTORS ``AS IS'' AND ANY
```

Comment:
Watch out, the first two quotes appear to be smart quotes, and it is throwing off GitHub's markdown parsing. I recommend converting all smart quotes to plain quotes.